### DISCUSSION TITLE
Are you hosting your media using a CDN?

### DISCUSSION THREAD

**Discussion by Programazing** [estruyf/vscode-front-matter#189] [Q&A] [answered]
I just started using Front Matter today and noticed the dashboard uses local media for the content previews.

I've been hosting my images on Cloudinary for the longest time because they give me better control over dynamic resizing and quality. That and I assumed using a CDN for images would be faster on a static site.

I'm curious if anyone else uses something similar. I didn't want to open an issue if I was the odd one out.

**Comment by zivbk1** [comment, marked as answer]
@Programazing we did discuss something like this in #97

**Comment by estruyf** [comment]
@Programazing as the first intention of Front Matter is to run locally without the need of an API/external service. It indeed makes a lot of sense to take it under consideration. 

The custom script support, mentioned in #97, is indeed part of the solution. If you have recommendations on how you see the integration with such services, let us know. I'll be more than happy to have this discussion.

Another idea is to somehow partner with one of these services to have a deeper integration that can be enabled while configuring the extension.

For my personal website, all files are returned via a CDN from Azure and Cloudflare btw.
